internal/commands/network/modify.go

```diff
@@ -39,11 +45,8 @@ func (s *modifyCommand) InitCommand() {
 		"  gateway: string \n"+
 		"  dhcp: true/false \n"+
 		"  dhcp-default-route: true/false \n"+
-		"  dhcp-dns: array of strings \n"+
-		"Usage \n"+
-		"	--ip-network dhcp-dns=<value1>,family=IPv4 \n"+
-		" --ip-network 'dhcp=true,\"dhcp-dns=<value1>,<value2>\",family=IPv6'")
-	s.AddFlags(fs) // TODO(ana): replace usage with examples once the refactor is done.
+		"  dhcp-dns: array of strings \n")
```

Comment:
is the tailing newline necessary? eg. does it create an empty line between the flag descriptions and if so, is that what we want?

Reply:
As far as I can tell I doesn't do anything (at least on mac, zsh). All those newlines in this description seem to not have any effect

Reply:
Interesting, good catch. Seems like something has changed somewhere at some point.. Ticketing this.